app/controllers/whitehall_media_controller.rb

```diff
@@ -1,14 +1,12 @@
 class WhitehallMediaController < BaseMediaController
-  include ActionView::Helpers::AssetUrlHelper
-
   def download
     path = "/government/uploads/#{params[:path]}.#{params[:format]}"
     asset = WhitehallAsset.find_by(legacy_url_path: path)
 
     if asset.unscanned?
       set_expiry(1.minute)
       if asset.image?
-        redirect_to image_path('thumbnail-placeholder.png')
+        redirect_to ActionController::Base.helpers.image_path('thumbnail-placeholder.png')
```

Comment:
I think this could be written more simply as:

```ruby
redirect_to self.class.helpers.image_path('thumbnail-placeholder.png')
```

I've checked this by running the following in the Rails console:

```
irb> class WhitehallMediaController < BaseMediaController
irb>   def foo
irb>     self.class.helpers.image_path('thumbnail-placeholder.png')
irb>   end
irb> end
=> :foo
irb> WhitehallMediaController.new.foo
=> "/asset-manager/thumbnail-placeholder-b0b645c0f44c2fc7d8cc2fee24f576d82256a85ac3c3e0a59d6ff60d2903d6b6.png"
```

Reply:
We've updated this to use `self.class.helpers` as you suggest.